snuba/cli/cleanup.py

```diff
@@ -15,6 +15,12 @@
 @click.option(
     "--clickhouse-port", type=int, help="Clickhouse native port to write to.",
 )
+@click.option(
+    "--clickhouse-user", type=int, help="Clickhouse user to login with.",
+)
+@click.option(
+    "--clickhouse-password", type=int, help="Clickhouse password to login with.",
+)
```

Comment:
Please let's not add user name and password to the CLI (for any command). This has two drawbacks:

- security. Adding passwords in command lines leaks them to whoever has access to the list of running processes on a system and to the shell history. Using settings and environment variables are better practices.
- Snuba configuration plan. We are actually removing Clickhosue connection settings from the command line everywhere and relying on settings instead.

Reply:
This comment is still valid. I see we are allowing now user and password to be set in settings, this is good, thanks.
Please let's not add the CLI arguments for these so we do not encourage the practice of passing them through the CLI. It is already possible to load the connection parameters from the settings. See line 74. 
All the connection CLI arguments will eventually be removed anyway.